src/shared/common/forms/form-repository.jsx

```diff
@@ -159,6 +160,12 @@ export function formComponentFactory(field, options) {
             return renderFormComponent(Inset, { key, data, config });
         case 'confirmation-with-case-ref':
             return renderFormComponent(ConfirmationWithCaseRef, { key, data, config, caseRef });
+        case 'confirmation-with-team-name-and-case-ref':
+            var teamName;
```

Comment:
any reason for the `var` over `const`?

Reply:
The intent was to allow the team name to be set based on any number of potential variables, not just for AcceptanceTeam, I could potentially move the setter and the return value inside of the if and use a const if this would be preferred?

Reply:
I think this is redundant if given the request below.